keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -855,8 +1069,8 @@ TODO: Identify more cases
 - ContainerStatus API changes are done. Tests are ready but not enforced.
 
 #### Beta
-- VPA alpha integration of feature completed and any bugs addressed,
-- E2E tests covering Resize Policy, LimitRanger, and ResourceQuota are added,
+- VPA alpha integration of feature completed and any bugs addressed.
```

Comment:
Hey, I'm just looking at this from the VPA side and I'm wondering if this line is making an integration into VPA a prerequisite for promoting the feature gate to beta?
The [project board](https://github.com/orgs/kubernetes/projects/178/views/1) has https://github.com/kubernetes/autoscaler/issues/4016 listed as "Post-GA", which indicates that this isn't a prerequisite.

Currently, there's no active development on getting the alpha version of in-place updates supported in VPA, so this is just a clarifying question if you're expecting progress to be made, before this feature graduates to beta.

Thanks!

Reply:
Ah, good catch. Thank you! We're hoping to land an initial VPA integration around the time this goes into beta, but I don't think it should block IPPR beta.